_posts/2021-08-23-gsoc-2021-project.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: post
+title: Google Summer of Code 2021 - Dask Project
+author: Freyam Mehta
+theme: twitter
+---
+
+{% include JB/setup %}
+
```

Comment:
We've been using an `## Executive Summary` heading for the short summary up the top.

The first sentence needs to be about why someone should read this post, eg: introducing new features

Also worth including is information about when & how people can use the new changes. Right now you can do that with the dask/distributed main branch, or wait for the next dask/distributed release (`2021.08.1` or higher).